src/game_launcher.cpp

```diff
@@ -575,8 +582,8 @@ game_launcher::unit_test_result game_launcher::single_unit_test()
 
 	try {
 		campaign_controller ccontroller(state_, game_config_manager::get()->terrain_types(), true);
-		LEVEL_RESULT res = ccontroller.play_replay();
-		if (!(res == LEVEL_RESULT::VICTORY) || lg::broke_strict()) {
+		ccontroller.play_replay();
```

Comment:
Why is it not necessary to check the return value anymore? If a failure guarantees `broke_strict` triggers then I’d say that warrants a comment.

Reply:
I removed it because a result that isn't victory doesn't necessarily mean there was a replay error, and also because no unit tests rely on that check.  If in the future there is a test that would rely on the LEVEL_RESULT, then a specific LEVEL_RESULT should be added to indicate that (and ideally also indicate what the type of problem was).